script/elastic/up.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# https://hub.docker.com/_/elasticsearch/
+
+
+sysctl -w vm.max_map_count=262144
+
+# use 1/2 of available memory
+avail_mem=`free -m | grep Mem | awk '{print $7}'`
+jvm_mem=$((avail_mem / 2))
+
+docker run -p 9200:9200 -p 9300:9300 \
+  -e ES_JAVA_OPTS="-Xms${jvm_mem}m -Xmx${jvm_mem}m" \
+  elasticsearch
```

Comment:
specify the version, same as influx and redis, though using latest is not that bad, but I don't want fetch the newest one, a working (most commonly used) one is enough